charts/prometheus-blackbox-exporter/templates/deployment.yaml

```diff
@@ -35,4 +35,13 @@ spec:
       {{- end }}
     spec:
     {{- include "prometheus-blackbox-exporter.podSpec" . | nindent 6 }}
+    {{- with .Values.podSecurityContext }}
```

Comment:
If I'm not mistaken, you are doing it twice now. The line above includes the helper, so you set securityContext twice on podLevel.